doc/vic20.sgml

```diff
@@ -219,8 +219,7 @@ The program return code (low byte) is passed back to BASIC by use of the
 
 <sect1>Using extended memory<p>
 
-The extended memory at $A000 may be added to the heap by using the following
-code:
+BLK5 memory may be added to the heap by using the following code:
```

Comment:
It's common practice, see [http://sleepingelephant.com/ipw-web/bulletin/bb/search.php?keywords=BLK5](url).